src/views/LimitedRestorationExtension.vue

```diff
@@ -265,14 +278,9 @@ export default class Restoration extends Vue {
       this.setResource(this.restorationResource)
 
       // initialize Fee Summary data
-      let filingData: FilingDataIF[] = []
-      if (this.isLimitedExtendRestorationFiling) filingData = [this.restorationResource.filingData[0]]
-      if (this.isLimitedConversionRestorationFiling) filingData = [this.restorationResource.filingData[1]]
-
-      filingData.forEach(fd => {
-        fd.futureEffective = this.getEffectiveDateTime.isFutureEffective
-      })
-      this.setFilingData(filingData)
+      // set the specific resource
+      this.setResource(this.restorationResource)
+      this.setFilingData([this.restorationResource.filingData])
```

Comment:
Simplifying.  Future effective isn't used for restoration filings.

Reply:
Please move comment on line 265 down to line 268.

Reply:
Good idea, I'll move the comment.

Reply:
I'm happy with this code now.